app/javascript/shared/components/emoji/EmojiInput.vue

```diff
@@ -93,59 +199,104 @@ $color-bg: #ebf0f5;
     position: absolute;
     right: $space-two;
   }
+}
+
+.emoji-list--wrap {
+  display: flex;
+  flex-direction: column;
+}
+
+.emoji--item {
+  background: transparent;
+  border: 0;
+  border-radius: $space-smaller;
+  cursor: pointer;
+  font-size: $font-size-medium;
+  height: $space-medium;
+  margin: 0;
+  padding: 0 $space-smaller;
+
+  &:hover {
+    background: $color-bg;
+  }
+}
+
+.emoji--row {
+  box-sizing: border-box;
+  padding: $space-smaller;
 
   .emoji--item {
-    cursor: pointer;
-    background: transparent;
-    border: 0;
-    font-size: $font-size-medium;
-    height: $space-medium;
-    border-radius: $space-smaller;
+    line-height: 1.5;
+    margin: $space-smaller;
+  }
+}
+
+.emoji-search--wrap {
+  background-color: $color-white;
+  margin: $space-small;
+  position: sticky;
+  top: $space-small;
+
+  .emoji-search--input {
+    background-color: $color-bg;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    font-size: $font-size-small;
+    height: $space-larger;
     margin: 0;
-    padding: 0 $space-smaller;
+    padding: $space-small;
+    width: 100%;
 
-    &:hover {
-      background: $color-bg;
+    &:focus {
+      box-shadow: 0 0 0 1px $color-woot, 0 0 2px 3px $color-primary-light;
     }
   }
+}
 
-  .emoji--row {
-    display: flex;
-    box-sizing: border-box;
-    height: 200px;
-    overflow-y: auto;
-    padding: $space-smaller;
-    flex-wrap: wrap;
+.empty-message {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: 212px;
+  justify-content: center;
 
-    .emoji--item {
-      margin: $space-smaller;
-      line-height: 1.5;
-    }
+  .emoji-icon {
+    color: $color-text-light;
```

Comment:
Css variables for colors are available for both the widget and dashboard. for other occurrences as well